### ISSUE TITLE
Black vertical and horizontal borders

### ISSUE THREAD

**Issue by mostafazakaria** [felipecsl/AsymmetricGridView#42]
I have a problems appears on android API <=16. The grid have Black vertical and horizontal borders. I tried to change gridview background and its parent also, but no thing changed

**Comment by mostafazakaria** [comment]
I have a problems appears on android API <=16. The grid have Black vertical and horizontal borders. I tried to change gridview background and its parent also, but no thing changed

![unnamed](https://cloud.githubusercontent.com/assets/12429999/9363911/973370a2-46a9-11e5-99c7-7e276a5b3bc1.png)

**Comment by shabbirsphinx** [comment]
I am also facing similar issue. @mostafazakaria you got any solution for this ? Any thread that would help me ?

**Comment by mostafazakaria** [comment]
@shabbirsphinx  Yes I made a workaround by adding two xml files to my project drawable folder
1) item_divider_horizontal.xml
`<?xml version="1.0" encoding="utf-8"?>
<shape xmlns:android="http://schemas.android.com/apk/res/android">
    <size android:width="3dip" />
    <solid android:color="@android:color/transparent"/>
</shape>`

2)item_divider_vertical.xml
`<?xml version="1.0" encoding="utf-8"?>
<shape xmlns:android="http://schemas.android.com/apk/res/android">
    <size android:height="3dip" />
    <solid android:color="@android:color/transparent"/>
</shape>`

This will override the default style which has black borders

**Comment by gregcabrera422** [comment]
How did you add spaces in between items?
